_posts/2020-05-01-issue-1.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: 'Issue 1: Solidarity Now More Than Ever'
+category: Volume 2
+image:
+    file: vol-2-issue-1-header.png
+    alt: The Tech Workers Coalition newsletter is back, with a special May Day edition!
+    caption: The Tech Workers Coalition newsletter is back, with a special May Day edition!
+    source_link: null
+    half_width: false
+---
+
+Today is May Day, also known as International Workers Day or [#EssentialWorkersDay](https://twitter.com/search?q=%23EssentialWorkersDay&src=typeahead_click). As shelter-in-place orders are extended in the Bay Area and beyond, it becomes increasingly clear just how much society relies on those who are unable to ‘shelter in place’: those working in supermarkets, or Amazon warehouses, or delivering goods to people’s homes. But even as their employers are celebrated for providing services to people in need, these frontline workers are rarely granted the respect they deserve -- not to mention the wages, benefits, or even protective equipment.
+
+<!--excerpt-->
+
+For many workers, the situation is intolerable, and they are taking a stand. Today, workers at Instacart, Shipt, Amazon warehouses, and your neighborhood Whole Foods are striking with a [list of demands](https://docs.google.com/document/d/1KD8uaVch14xBD1TTZ-TKbAub-IcqERS6O1OVeL2HbtY/edit). Christian Smalls, Amazon warehouse worker and organizer who was fired earlier this month, shared a [video montage](https://twitter.com/Shut_downAmazon/status/1255681771292721158) featuring dozens of essential workers who are walking out to protest for better working conditions.
+
+{% include post_image.html %}
+
+## THE WORKER’S PERSPECTIVE
```

Comment:
I'm a fan of lowercase for all of these. Anyone else?

Reply:
yep make sense (i just copied from the doc), will do that now